src/main/java/io/strimzi/kafka/bridge/quarkus/CustomMeterRegistryProducer.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+package io.strimzi.kafka.bridge.quarkus;
+
+import io.micrometer.core.instrument.Meter;
+import io.micrometer.core.instrument.config.MeterFilter;
+import io.micrometer.prometheus.PrometheusConfig;
+import io.micrometer.prometheus.PrometheusMeterRegistry;
+import io.micrometer.prometheus.PrometheusNamingConvention;
+
+import javax.annotation.PostConstruct;
+import javax.enterprise.inject.Produces;
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+/**
+ * Used for scraping and reporting metrics in Prometheus format
+ */
+@Singleton
+public class CustomMeterRegistryProducer extends PrometheusMeterRegistry {
+
+    @Inject
+    JmxCollectorRegistry restJmxCollectorRegistry;
+
+    @PostConstruct
+    void init() {
+        this.config().meterFilter(
+                MeterFilter.deny(meter -> "/metrics".equals(meter.getTag("uri"))));
+        this.config().namingConvention(new CustomMeterRegistryProducer.MetricsNamingConvention());
+    }
+
+    /**
+     * Came in as part of extending `PrometheusMeterRegistry since
+     * it has no default constructors available
+     */
+    private CustomMeterRegistryProducer(PrometheusConfig config) {
+        super(config);
+    }
+
+    @Produces
+    @Singleton
+    CustomMeterRegistryProducer createPrometheusMeterRegistry() {
+        return this;
+    }
+
+    /**
+     * Scrape metrics on the provided registries returning them in the Prometheus format
+     *
+     * @return metrics in Prometheus format as String
+     */
+    @Override
+    public String scrape() {
+        StringBuilder sb = new StringBuilder();
+        if (Boolean.parseBoolean(System.getenv("KAFKA_BRIDGE_METRICS_ENABLED"))) {
```

Comment:
can we try the Quarkus way ... something like this declaring a bean field:

```java
@ConfigProperty(name = "kafka.bridge.metrics.enabled", defaultValue = "false")
boolean isMetricsEnabled;
```

it should work as property in the application.properties file as well as by setting or not the KAFKA_BRIDGE_METRICS_ENABLED env var.

Reply:
Iam trying this but I wonder why the value is being setted as `true`

Reply:
> Iam trying this but I wonder why the value is being setted as true

When you don't set the env var? is it true?

Reply:
Yeah, even when I set the `defaultValue` as false its being evaluated as true